shopyo/modules/box__bizhelp/announce/templates/announce/list.html

```diff
@@ -3,25 +3,58 @@
 {% block pagehead %}
 <title></title>
 <style>
+
 </style>
 {% endblock %}
 {% block sidebar %}
 {%include info['module_name']+'/blocks/sidebar.html'%}
 {%endblock%}
 {% block content %}
+<script>
+$(function() {
+    $('.delete_me').click(function() {
+        var id = $(this).attr('id');
+        console.log(id);
+        var announce_id = id.split('-')[1];
+        var announce_title = id.split('-')[2];
+        console.log(announce_id);
+        var ask = confirm("Do you want to delete "+announce_title+" ?");
+        if (ask === true) {
+            window.open("{{info['url_prefix']}}/" + announce_id + "/delete/check", "_self")
+        } else if (ask === false) {
+
+        }
+    });
+});
+</script>
 <br>
 <div class="card">
     <div class="card-body">
-        {%for announce in announcements%}
-            <div class="card">
-                <div class="card-header">
-                    {{announce.title}} | {{announce.created_date.strftime("%m/%d/%Y, %H:%M:%S")}}
-                </div>
-                <div class="card-body">
-                    {{announce.content | safe}}
+        <div class="row">
+            {%for announce in announcements%}
+            <div class="col-sm-3 col-xs-6 col">
+                <div class="card annnounce--listed-card">
+                    <div class="card-header">
+                        {{announce.title}} | {{announce.created_date.strftime("%m/%d/%Y, %H:%M:%S")}}
+                    </div>
+                    <div class="card-body">
+                        {{announce.content | safe}}
+                    </div>
+                    <div class="card-footer">
+                        <div class="btn-group" role="group" aria-label="Basic example">
+                          <a href="{{url_for(info['module_name']+'.edit', announce_id=announce.id)}}"type="button" class="btn btn-warning">
+                            <i class="fa fa-pencil-alt"></i>
+                        </a>
+                          <a href="#"type="button" class="btn btn-danger delete_me" id="delete-{{announce.id}}-{{announce.title}}">
+                            <i class="fa fa-trash-alt text-white"></i>
+                          </a>
+                        </div>
+                    </div>
```

Comment:
Having multiple announcement does not looks good if the content is large. Better to just have one announcement take up all col (or at least allow only half the col i.e col-6 class). Another option is to add ellipsis for long content and show more button etc. I am not sure what the best approach is here as this is more of a UI design decision

![ann1](https://user-images.githubusercontent.com/5404500/104793607-745b6600-5771-11eb-8cf9-ca0b915a30dd.PNG)
![image](https://user-images.githubusercontent.com/5404500/104793592-5ee63c00-5771-11eb-9d95-661f3ca5543b.png)

Reply:
Will fiiiix

Reply:
Works fine for me

![textissue](https://user-images.githubusercontent.com/22630684/104806078-449e7380-57ee-11eb-9a92-9d68aacc2e5e.gif)

Reply:
Yeah it works but just didn't look good. Not an issue for now.